index.js

```diff
@@ -432,11 +415,20 @@ class ClientServiceBase {
    * @return {Promise<CertificateRegistrationRequest>}
    */
   async _createCertificateRegistrationRequest() {
+    const properties = new ClientProperties(
+        this.properties,
+        [
+          'scalar.dl.client.cert_pem',
```

Comment:
It is possible to use the constants defined in `client_properties.js` instead ?

Reply:
Ah.
Yes. I should use the constants outside too.

Reply:
Great, thanks